_pkgdown.yml

```diff
@@ -2,11 +2,6 @@ url: https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/
 
 template:
   bootstrap: 5
-  bootswatch: cyborg
-  theme: arrow-dark
-  bslib:
-    primary: "#0054AD"
-    border-radius: 0.5rem
-    btn-border-radius: 0.25rem
-
 
+development:
+  mode: auto
```

Comment:
removed dark theme and added developemnt auto mode https://pkgdown.r-lib.org/reference/build_site.html#development-mode

The site was built then under docs/dev.
All other changes are the built site